frontend/src/metabase/entities/collections/constants.ts

```diff
@@ -1,12 +1,17 @@
 import { t } from "ttag";
 
+import { Collection } from "metabase-types/api";
+
 export const DEFAULT_COLLECTION_COLOR_ALIAS = "brand";
 
-export const ROOT_COLLECTION = {
+export const ROOT_COLLECTION: Collection = {
```

Comment:
We shouldn't do it like that. The constant is used in the app for real and setting values `can_write: false` affects the runtime behavior. Furthermore, it's on purpose not a `Collection`, it's only a constant with a set of properties that the root collection has. 

If you need to pass it to somewhere in test where a `Collection` is expected, you need to use `createMockCollection(ROOT_COLLECTION)`